wifiphisher/common/accesspoint.py

```diff
@@ -24,25 +26,116 @@ def __init__(self):
         self.force_hostapd = False
         # roguehostapd object
         self.hostapd_object = None
+        self.deny_mac_addrs = []
+        self.dns_conf_path = constants.DNS_CONF_PATH
+
+    def enable_system_hostapd(self):
```

Comment:
We have removed these since your PR was introduced: https://github.com/wifiphisher/wifiphisher/commit/b8e3897e76ff571ecff87213f3e04df5e68548ae#diff-b4ce7f5760590e5f3d95a88c1e9223a3


Can you remove them from your PR?